components/accordion/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 6.0.0
+
+### Minor Changes
+
+- [#2616](https://github.com/adobe/spectrum-css/pull/2616) [`7f45ea9`](https://github.com/adobe/spectrum-css/commit/7f45ea95d3d31addf29b0720de8623b0f3f0431d) Thanks [@castastrophe](https://github.com/castastrophe)! - ## Build optmizations to support minification
+
+  Output for all component CSS files is now being run through a lightweight optimizer (cssnano) which significantly reduces unnecessary whitespace. These changes reduce file size but should not have any impact on the rendering of the component. By removing unnecessary whitespace from var functions, we are making it easier to effectively minify our provided CSS assets.
+
+### Patch Changes
+
+- Updated dependencies [[`7f45ea9`](https://github.com/adobe/spectrum-css/commit/7f45ea95d3d31addf29b0720de8623b0f3f0431d)]:
```

Comment:
@pfulton This feels like a bug - it's reading peer dependency changes as a dependency update and thus forcing a major bump when I identified this update as a minor bump.

Reply:
https://github.com/changesets/changesets/blob/main/docs/decisions.md#the-versioning-of-peer-dependencies